assets/rich_text_components_definitions.ts

```diff
@@ -112,20 +112,20 @@ export = {
       "default_value": ""
     }]
   },
-  "ConceptCard": {
-    "backend_id": "concept-card",
+  "Conceptcard": {
```

Comment:
I think I understand why you did this, but it does look a little bit weird. How about using "Concept" throughout?

Reply:
Done